sample_usage/stats_analysis.py

```diff
@@ -72,21 +73,28 @@ def summary(all_msgs):
     q3 = stu_errors[ceil(0.25 * len(stu_errors)) - 1]
     q1 = stu_errors[ceil(0.75 * len(stu_errors)) - 1]
 
+    # Calculating the Standard Deviation
+    sum_dev = 0
+    for error_dev in stu_errors:
+        sum_dev += (error_dev - avg_errors) ** 2
+
+    std_dev = sqrt(sum_dev / len(stu_errors))
+
     sum_stats = [('Most Frequent Code Errors', error_num),
                  ('Average Code Errors Per Student',
                   round(len(code_errors) / num_stu, 2)),
                  ('Most Frequent Style Errors', style_num),
                  ('Average Style Errors Per Student',
                   round(len(style_errors) / num_stu, 2)),
                  ('Most Frequent Errors of Either Type', both_num),
-                 ('Average Errors of Either Type Per Student',
-                  round((len(code_errors) + len(style_errors)) / num_stu, 2)),
+                 ('Average Errors of Either Type Per Student', avg_errors),
                  ('Five Number Summary of Errors Per Student',
                   [('Most Errors', stu_errors[0]),
                    ('Upper Quartile (Q3)', q3),
                    ('Median', med),
                    ('Lower Quartile (Q1)', q1),
-                   ('Least Errors', stu_errors[-1])])]
+                   ('Least Errors', stu_errors[-1])]),
+                 ('Standard Deviation', round(std_dev, 2))]
```

Comment:
sorry, what do you mean? Standard Deviation is under another category, not under Five Number Summary of Errors Per Student